app/views/account/users/_form.html.slim

```diff
@@ -0,0 +1,8 @@
+div
+  = simple_form_for(@user, url: path) do |f|
+    = f.input :name
+    = f.input :email
+    = f.input :password
+    = f.input :password_confirmation
+    = f.button :submit
```

Comment:
This form is not used after we decided that admin could not create moderators. As we discussed above, we will return it if we decide to edit something in user profile.